examples/config/config

```diff
@@ -321,6 +329,7 @@ set history_disable_easter_egg 1
 # Set function shortcut
 @cbind  s<var:>_<value:>_  = set %1 %2
 # Exit binding
+@cbind  q                  = exit
```

Comment:
Do you imagine people having that problem, or do you actually hit keys accidentally yourself? (seriously curious)